site/index.html

```diff
@@ -1,5 +1,34 @@
 <html>
-  <body>
-  </body>
-  <script src="bundle.js"></script>
+    <head>
+        <title>Match 3 - Home</title>
+        <link rel="stylesheet" type="text/css" href="stylesheet.css">
+    </head>
+    <body>
+        <header>
+            <a href="index.html" id="match3">Match 3</a>
+        </header>
+        <div class="right">
+            <a href="demo.html">Demo</a><br>
+            <a href="docs.html">Documentation</a>
+        </div>
+        <div class="content">
+            <p>An engine for driving <a href="https://en.wikipedia.org/wiki/Category:Match_3_games">a "match three" game</a>, written in Javascript. Allows for plenty of customization, but defaults to mimic the match three system found in <a style="font-style: italic;" href="https://www.youtube.com/watch?v=GGWdqf1BnXo&t=13s">Gems of War</a>. This app is meant to be used as a core engine for a match three game, meaning that it contains no business logic concerning turns, or stats. This module only handles the moving, matching, removing, replenishing, and shuffling of "orbs". Orbs are the units that populate the board, and must be matched in groups of three or more to be removed from the board.</p>
+            <h2>Features</h2x>
```

Comment:
`h2x`

Eh, this whole thing should be auto-generated. Good idea, but better to just leave it as a link. Otherwise someone's going to have to keep the two documents in sync. That's no fun.